.github/workflows/deploy.yaml

```diff
@@ -71,17 +77,16 @@ jobs:
             jq -r 'to_entries[] | [.key, .value] | @tsv')
 
       - name: create npmrc
```

Comment:
Node version in our image is 20.12.2, GHA's is 20.19.0. Close enough. I think package.json should cover the rest and since this is where Typescript comes from, probably more important, so this seems OK